index.html

```diff
@@ -0,0 +1,813 @@
+                            justo
+                            imperdiet, wisi risus purus auge
+                            vulputate
+                            voluptate neque, curabitur dolor libero sodales vitae elit massa</p>
+                    </div>
+                </div>
+                <div class="service__item">
+                    <div class="glyph">
+                        <div class="fs1" aria-hidden="true" data-icon="&#xe027;"></div>
+                    </div>
+                    <div class="service-text">
+                        <h1 class="service-text__title">CMS & eCommerce</h1>
+                        <p class="service-text__description">Lorem ipsum dolor sit amet, maecenas eget vestibulum
+                            justo
+                            imperdiet, wisi risus purus auge
+                            vulputate
+                            voluptate neque, curabitur dolor libero sodales vitae elit massa</p>
+                    </div>
+                </div>
+            </div>
+
+        </section>
+        <section id="about" class="section-gray section-about">
+            <div class="text-left">
+                <h1>Genius Studio</h1>
+                <p>Lorem ipsum dolor sit amet, maecenas eget vestibulum
+                    justo
+                    imperdiet, wisi risus purus auge
+                    vulputate
+                    voluptate neque, curabitur dolor libero sodales vitae elit massa.Lorem ipsum dolor sit amet,
+                    maecenas
+                    eget vestibulum
+                    justo
+                    imperdiet, wisi risus purus auge
+                    vulputate
+                    voluptate neque, curabitur dolor libero sodales vitae elit massa.<br> Lorem ipsum dolor sit amet,
+                    maecenas eget vestibulum
+                    justo
+                    imperdiet, wisi risus purus auge
+                    vulputate
+                    voluptate neque, curabitur dolor libero sodales vitae elit massa</p>
+            </div>
+            <iframe class="about__video" width="330" height="180" src="https://www.youtube.com/embed/2ZKV6B03I0s"
+                frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+        </section>
+        <section id="team-members" class="section-white">
+            <div class="text-center">
+                <h1>team member</h1>
+                <p> Lorem ipsum dolor sit amet, maecenas eget vestibulum
+                    justo
+                    imperdiet</p>
+            </div>
+            <div class="team">
+                <div class="team-member">
+                    <div class="team-image__container">
+                        <img src="images/team_1.jpg" alt="Johnathan">
+                        <div class="team-contact__icons">
```

Comment:
recommend avoiding double underscore in class names unless you are following BEM (all the file follows the strict rules of BEM)